src/tests/unit/tests/common/components/cards/instance-details-group.test.tsx

```diff
@@ -16,12 +17,14 @@ import { exampleUnifiedRuleResult } from './sample-view-model-data';
 describe('InstanceDetailsGroup', () => {
     it('renders', () => {
         const fixInstructionProcessorMock = Mock.ofType(FixInstructionProcessor);
+        const recommendColorMock = Mock.ofType(RecommendColor);
```

Comment:
Since you aren't simulating any behavior here, this should just be a stub:
```suggestion
        const recommendColorStub = {} as RecommendColor;
```